StratisCore.UI/src/app/shared/pipes/seconds-to-string.pipe.ts

```diff
@@ -0,0 +1,48 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'secondsToString'
+})
+export class SecondsToStringPipe implements PipeTransform {
+  constructor() {
+  }
+
+  public transform(seconds: number): string {
+    let numDays = Math.floor(seconds / 86400);
+    let numHours = Math.floor((seconds % 86400) / 3600);
+    let numMinutes = Math.floor(((seconds % 86400) % 3600) / 60);
+    let numSeconds = ((seconds % 86400) % 3600) % 60;
```

Comment:
not used?

Reply:
It's used in the staking.component.html file

Reply:
I am referring to `numSeconds` variable. It comes up as unused, how does html use a locally scoped variable?